core/src/main/scala/akka/persistence/cassandra/query/scaladsl/CassandraReadJournal.scala

```diff
@@ -152,6 +143,19 @@ class CassandraReadJournal(system: ExtendedActorSystem, cfg: Config)
       .map(_.setConsistencyLevel(queryPluginConfig.readConsistency).setIdempotent(true)
         .setRetryPolicy(readRetryPolicy))
 
+  // FIXME consider only allowing Q/LQ consistnecy for all tag related things
```

Comment:
yes, I also think we should enforce sane consistency levels, not only for this, also for the write side

Reply:
separate ticket

Reply:
https://github.com/akka/akka-persistence-cassandra/issues/258